src/components/footer/Footer.module.scss

```diff
@@ -1,38 +1,109 @@
-.footer {
-  min-height: 52px;
-  height: 8vh;
-  width: 100vw;
+.gtFooter {
+  height: 100%;
+  min-height: 60px;
   background-color: var(--gt-secondary);
+  display: flex;
+  align-items: center;
 }
 
-.container{
+.footer{
   height: 100%;
-  width: 100%;
+  //width: 100%;
   display: flex;
-  align-items: center;
-  justify-content: space-between;
+  flex-direction: column;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
 }
 
-.leftSection{
-  width: 25%;
+
+.footer .centerSection{
+  display: flex;
+  justify-content: center;
+  order: 1;
 }
 
-.centerSection{
-  width:100%;
-  text-align: center;
+.footer .centerSection p{
+  margin: 0;
 }
 
-.centerSection p{
-  margin-bottom: 0;
+.footer button{
+  width: 100%;
+  margin-bottom: 1rem;
 }
 
-.rightSection{
-  width:25%;
-  display: flex;
-  justify-content: right;
+
+@media (min-width: 600px) {
+  .footer{
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .footer button{
+    width: auto;
+    margin:0;
+  }
+  .footer .centerSection{
+    order: 0;
+  }
+  .footer .rightSection{
+    display: flex;
+    column-gap: 0.5rem;
+    justify-content: space-between;
+  }
+  .footer .rightSection button{
+    //width: 5rem;
+  }
 }
-.rightSection button{
-  margin: 0 0 0 10px;
+
+
+@media (min-width: 960px) {
+
+  .footer .leftSection{
+    justify-content: left;
+  }
+  .footer .xd{
```

Comment:
.xd isnt used anywhere in this app, this selector can be deleted

Reply:
xd